networkx/readwrite/json_graph/tests/test_cytoscape.py

```diff
@@ -10,7 +10,7 @@
 def test_graph():
     G = nx.path_graph(4)
     H = cytoscape_graph(cytoscape_data(G))
-    nx.is_isomorphic(G, H)
+    assert nx.is_isomorphic(G, H)
```

Comment:
So in this file it's still `is_isomorphic`... is that because there are serialization differences w/ the cytoscape functions? Or can we use `graphs_equal` here as well?

Reply:
Cytoscape is also not removing the serialisation artifacts, although I hadn't noticed until I looked (I just added the asserts). I will fix in an upcoming patchset.

def test_example():
    G = DiGraph()
    G.add_edge("A", "B")
    H = cytoscape_graph(cytoscape_data(G))
    print(H.edges[("A", "B")])  # {"source": "A", "target": "B"}
    print(G.edges[("A", "B")])  # {}

Reply:
It seems that Cytoscape serialisation isn't reliably deserialisable without adding metadata to the serialised form, and I am not familiar with the schema for Cytoscape.

Similarly as above:

def test_example():
  G = DiGraph()
  G.add_edge("A", "B")
  H = cytoscape_graph(cytoscape_data(G))
  print(H.nodes["A"]) # {'id': 'A', 'name': 'A'}
  print(G.edges["A"]) # {} 

But I can't blanket remove the "id" and "name", as they could be valid:


def test_example():
  G = DiGraph()
  G.add_edge("A", "B")
  G.add_node("A", name="A", id="A")
  H = cytoscape_graph(cytoscape_data(G))
  print(H.nodes["A"]) # {'id': 'A', 'name': 'A'}
  print(G.nodes["A"]) # {'id': 'A', 'name': 'A'}

Reply:
Ah, Barnowski is you, so you may have input